source/common/config/external_config_validators.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "envoy/config/config_validator.h"
+
+namespace Envoy {
+namespace Config {
+
+// Represents a collection of external config validators for all xDS
```

Comment:
What is "external" here?

Reply:
I've updated the comment here, let me know if you think it should be further clarified.

Reply:
Changed the name to Custom Config Validators.